internal/pkg/filesystem/aferofs/mountfs/mountfs.go

```diff
@@ -74,13 +75,13 @@ func (v *MountFs) SubDirFs(path string) (interface{}, error) {
 	// Copy mount points
 	var mounts []MountPoint
 	for _, m := range v.mounts {
-		if mountPath, err := filesystem.Rel(path, m.Path); err == nil {
+		if mountPath, err := filesystem.Rel(v.ToSlash(path), m.Path); err == nil && mountPath != "" {
 			mounts = append(mounts, MountPoint{mountPath, m.Fs})
 		}
 	}
 
 	// Create filesystem
-	basePath := filesystem.Join(v.BasePath(), path)
+	basePath := filepath.Join(v.BasePath(), path) // nolint: forbidigo
```

Comment:
Posledny commit su fixes, aby to islo na Windows.

Cesty k `mountpoints` su z high level API, ktore pouziva vzdy `/` ako oddelovac.
No operacie nad subormi su z afero libky, a ta pouziva oddelovac podla OS.
A preto je potrebne na danych miestach paths skonvertovat.
Aby to islo, a aby aj na Windows boli vsetky chybove hlasky s `/` ako oddelovacom v path.